docs/design/coreclr/profiling/davbr-blog-archive/Profiler stack walking Basics and beyond.md

```diff
@@ -77,50 +77,50 @@ Before I continue from this exciting cliffhanger, a brief interlude.  Everyone k
 
 Now that we’re speaking the same language.  Let’s look at a mixed-mode stack:
 
-| 
+|
 
 Unmanaged
 
  |
-| 
+|
```

Comment:
These appear in the same line (both before and after the change), is that desired?
I couldn't guess what is the desired output here.

Reply:
I am not able to find the formatting from the original blog post. I think formatting it like code block would communicate the point in the most readable way:

```
Unmanaged
D (Managed)
Unmanaged
C (Managed)
B (Managed)
Unmanaged
A (Managed)
Main (Managed)
```